src/utils/response.ts

```diff
@@ -0,0 +1,17 @@
+import { Response } from 'express';
+
+const sendResponse = <T>(
+  res: Response,
+  status: number,
+  data: T,
+  message: string
+): Response => {
+  const result = /^20\d$/.test(status.toString()) ? 'Success!' : 'Error!';
+  return res.status(status).json({
+    status: result,
+    data,
+    message,
+  });
+};
+
+export default sendResponse;
```

Comment:
good work @TuyishimireEric on this helper

Reply:
Thank you 🙏